veps/sig-compute/hypervisor-abstraction.md

```diff
@@ -81,6 +82,47 @@ spec:
 - `virt-controller` reads the configured hypervisor from `ClusterConfig` when generating launcher manifests and threads that ID through the `ConverterContext` so downstream components can act consistently.
 - Each package's registry uses the configured name to locate its implementation, avoiding a monolithic factory while keeping selection logic consistent.
 
+### Capability Registry Model
+
+The capability registry provides a declarative, structured way to express feature support:
+
+```go
+// Core types in pkg/capabilities/types.go
+type CapabilityKey string  // e.g., "graphics.vga", "firmware.secureboot.uefi"
+type SupportLevel int      // Supported, Unsupported, Experimental, Deprecated
+
+type Capability struct {
```

Comment:
Sounds more like CapabilitySupport. As for the very same capability CapGraphicsVGA these values will be different for kvm/amd64 and kvm/arm64